capstone/src/main/java/com/google/step/clients/MetricsClient.java

```diff
@@ -113,6 +113,27 @@ public WeeklyPageView getCurrentPageViews(String restaurantKey) {
         return pageViewsList;
     }
 
+    /**
+     * Private class to better organize the necessary information about a restaurant.
+     */
+    private class RestaurantPageViewInformation {
```

Comment:
`private static class`

If you don't declare the nested class as static, each instance will have a hidden reference to the parent. This can have garbage collection implications.

See Item 24 in effective java https://effectivejava.googleplex.com/third-edition

Reply:
Changed!